frontend/src/models/study.ts

```diff
@@ -107,6 +116,41 @@ export function getStudyDuration(study: ParticipantStudy): number {
     return sumBy(study.stages, (s) => +(s.durationMinutes || 0))
 }
 
+export const useAdminGetStudies = (status: string = 'all') => {
+    const api = useApi()
+    return useQuery('adminGetStudies', async () => {
+        const res = await api.adminQueryStudies({ status });
+        return res.data || [];
+    })
```

Comment:
Great point - i did a little more digging on this, seems like its a thing with react query, lots of discussion around it:

I think your suggestion [lines up with this discussion](https://github.com/TanStack/query/discussions/1301#discussioncomment-637669)

Just below that someone offers [an elegant solution](https://github.com/TanStack/query/discussions/1301#discussioncomment-2326904), maybe avoiding all the potential typescript issues noted in the above issue? ^

Seems like the placeholderData prop "should" work but it doesn't work because of [this](https://github.com/TanStack/query/discussions/1301#discussioncomment-1501870)

the alternative mentioned in that thread might be to check the `isSuccess` value from useQuery, which does work - however its another check in the components (which might be useful in certain scenarios, but for lists of data i think the below should work great)

```
    const { data: banners = [], isLoading } = useFetchBanners()

```